src/platform/posix.c

```diff
@@ -72,8 +72,13 @@ void platform_get_date_utc(rct2_date *out_date)
 	time(&rawtime);
 	timeinfo = gmtime(&rawtime);
 	out_date->day = timeinfo->tm_mday;
-	out_date->month = timeinfo->tm_mon;
-	out_date->year = timeinfo->tm_year;
+	if (OPENRCT2_PLATFORM == "Windows") {
```

Comment:
posix.c is not used by Windows, so you don't need to do this.

Also in the future the correct way to do this is `#ifdef _WINDOWS_`